src/utils/fetches.js

```diff
@@ -1,18 +1,71 @@
+import { getToken } from './storage';
+
 export const fetchPost = (api, obj) => {
+  const token = getToken();
+
   return fetch(api, {
     method: 'POST',
     headers: {
       'Content-Type': 'application/json',
+      Authorization: token,
```

Comment:
```suggestion
      Authorization: getToken(),
```
굳이 token을 변수에 한번 저장했다가 넣을 필요 없어보입니다. 함수 이름만으로도 리턴하는 값이 무엇인지 추론할 수 있을거 같아서요.

Reply:
현찬님과 의논하여 공용으로 넣기로 하였습니다